src/transformers/models/whisper/generation_whisper.py

```diff
@@ -438,21 +482,33 @@ def generate(
                 specific kwargs should not be prefixed and decoder specific kwargs should be prefixed with *decoder_*.
 
         Return:
-            [`~utils.ModelOutput`] or `torch.LongTensor` or `Dict[str, Any]`: A [`~utils.ModelOutput`] (if `return_dict_in_generate=True`
-            or when `config.return_dict_in_generate=True`) or a `torch.FloatTensor` or a dict of segments when `return_segments=True`.
+            [`~utils.ModelOutput`] or `Dict[str, Any]` or `torch.LongTensor`:
 
-                If the passed input is > 30 seconds / > 3000 mel input features and `return_segments=True` then a dictionary of generated sequence ids, called `sequences` and a list of each generated segment is returned.
+                A [`~utils.ModelOutput`] (when `return_dict_in_generate=True` and the passed input requires only one call to the underlying `generate` method) or a `Dict[str, Any]` (when `return_segments=True` or `return_token_timestamps=True`, with keys `sequences`, `segments` if `return_segments=True`, `token_timestamps` if `return_token_timestamps=True`;
+                `sequences` and `token_timestamps` includes the decoder input ids and end of sequence id if the passed input requires only one call to the underlying `generate` method) or a torch.LongTensor` (including the decoder input ids and end of sequence id if the passed input requires only one call to the underlying `generate` method).
```

Comment:
I think this part would benefit from being clearer: don't hesitate to use bullet points and spaces, and avoid parentheses.

Reply:
fixed in [0ea240f](https://github.com/huggingface/transformers/pull/34135/commits/0ea240fa88d2c7aeb3b8968427fb3f15a8e76e15) 👌